command/v7/shared/app_summary_displayer.go

```diff
@@ -159,6 +161,12 @@ func (display AppSummaryDisplayer) displayProcessTable(summary v7action.Detailed
 		}
 		display.displayAppInstancesTable(process)
 	}
+
+	if summary.Deployment.StatusValue == constant.DeploymentStatusValueActive {
```

Comment:
**suggestion**

The output currently looks like:
```
type:           web
sidecars:
instances:      2/2
memory usage:   1024M
     state     since                  cpu    memory        disk           logging          cpu entitlement   details
#0   running   2024-07-23T14:58:09Z   0.0%   48.8M of 1G   130.2M of 1G   22B/s of 16K/s   0.0%
#1   running   2024-07-23T14:58:14Z   0.0%   0B of 1G      0B of 1G       0B/s of 0B/s     0.0%

type:           worker
sidecars:
instances:      0/0
memory usage:   1024M
There are no running instances of this process.
Rolling deployment currently DEPLOYING
```

I think it would be nice to include some new lines for readability after the worker section

Reply:
Related, but also putting the deployment message by the web process that is deploying would also be an option. That feels more correct, but then perhaps it hides that message in the middle of the output. 🤔

Reply:
Will add new line. Proposal was to add it at the end, so kept it there. There is also a period at the end of message, will add that as well

Reply:
Done